src/Action/PlaceholderAction.php

```diff
@@ -25,7 +25,7 @@ final class PlaceholderAction
      *
      * @return object
      */
-    public function __invoke($data)
+    public function __invoke(mixed $data = null)
```

Comment:
```suggestion
    public function __invoke($data = null)
```

For compatibility with PHP 7. Change the PHPDoc instead.

Reply:
Done in commit eada6c0bc00762ec50a0c4a2b9889df3bcbf65ae